ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -396,7 +396,11 @@ fi
 # Benewake TFMini
 if param greater SENS_EN_TFMINI 0
 then
-	tfmini start
+	if ver hwcmp PX4FMU_V3
+	then
+		# start the driver on serial 4/5
+		tfmini start -d /dev/ttyS6
```

Comment:
Can we do this more generically? There are way too many variants now to maintain these special cases.

Reply:
I know... But I can't think of anything... On every board, it's different and we would need to link it to https://github.com/ChristophTobler/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS#L641-L672 as on most boards need to use this device path.

Reply:
Fair enough, I'll see if I can come up with something in the near future. https://github.com/PX4/Firmware/issues/8302

Reply:
@ChristophTobler So based on this, is the [documentation here](https://docs.px4.io/en/sensor/rangefinders.html#tfmini) accurate? It looks like the TFmini is only automatically started on Pixhawk FMUv3 (but not before or later) ... right?

If that is the case then should we not link to instructions for enabling on other boards - i.e. instructions on how to add a driver to a config and how to start/stop the driver and specify the port to watch? 

@dagar 
- Will the changes you propose in https://github.com/PX4/Firmware/issues/8302 make enabling sensors consistent and generic? I'm envisaging that you would set SENS_EN_TFMINI with the value of the port you wanted to use ... and ideally that this setting would automatically disable any other thing that is trying to use the port.
- Do you know what we need to do to create module docs ([like this](https://dev.px4.io/en/middleware/modules_driver.html) for tfmini? That would allow me to have somewhere to link to for "using the driver" instructions.

Reply:
I wasn't sure how to structure it. Configuring a parameter per serial port is a little more straightforward, but it's less modular. Parameters to enable per device are relatively easy, and keeps it separate, but what happens if you configure a conflict? Who takes priority?

I believe all of that documentation is generated from the module itself and Tools/px_process_module_doc.py https://github.com/PX4/Firmware/blob/master/src/drivers/px4fmu/fmu.cpp#L3394

Reply:
@dagar Plug'n'play would be nice :-). I understand your dilemma and there is no easy solution. My "docs" point of view is that it doesn't matter as long as we're completely consistent. Personally though I'd go for the less-modular per-serial port option because debugging conflicts would be a pain.  Is there a way to reduce the modularity issues - I'm thinking using toolchain to build per-port mappings from driver directories (maintaining current lists).

With respect to docs, I believe @bkueng new module shows what would need to be added for this module docs: https://github.com/PX4/Firmware/blob/master/src/templates/module/module.cpp
Anything else need to be done Beat?

@ChristophTobler As [above](https://github.com/PX4/Firmware/pull/8726#discussion_r164316951), please confirm the documentation is accurate and confirm the FMU versions this starts "out of the box" for.

Reply:
Cheers, I have added you as reviewer for this update in https://github.com/PX4/px4_user_guide/pull/138